python-checks/src/main/java/org/sonar/python/checks/ModifiedParameterValueCheck.java

```diff
@@ -160,27 +181,22 @@ private static Optional<PythonQuickFix> getQuickFix(FunctionDef functionDef, Exp
     return Optional.ofNullable(parameterInitialization(defaultValue)).map(
       paramInit -> PythonQuickFix.newQuickFix("Initialize this parameter inside the function/method")
         .addTextEdit(replace(defaultValue, "None"))
-        .addTextEdit(insertLineBefore(firstStatement, String.format("if %1$s is None:\n    %1$s = %2$s()", paramName, paramInit)))
+        .addTextEdit(insertLineBefore(firstStatement, String.format("if %1$s is None:\n    %1$s = %2$s", paramName, paramInit)))
         .build()
     );
   }
 
   @CheckForNull
   private static String parameterInitialization(Expression defaultValue) {
     if (defaultValue.is(CALL_EXPR)) {
-      CallExpression call = (CallExpression) defaultValue;
-      if (!call.arguments().isEmpty()) {
-        return null;
```

Comment:
it looks like we're changing the behaviour here. Before we weren't suggesting a quickfix if the call had arguments. I'm not sure what was the rationale behind this. 
Did you this intentionally?

Reply:
Cause there was no implementation to render string from the Tree model of the assigned value. Now it is implemented so we can support function calls with arguments